driver/esp32/ili9XXX.py

```diff
@@ -97,7 +97,8 @@ class ili9XXX:
     def __init__(self,
         miso=5, mosi=18, clk=19, cs=13, dc=12, rst=4, power=14, backlight=15, backlight_on=0, power_on=0,
         spihost=esp.HSPI_HOST, spimode=0, mhz=40, factor=4, hybrid=True, width=240, height=320, start_x=0, start_y=0,
-        invert=False, double_buffer=True, half_duplex=True, display_type=0, asynchronous=False, initialize=True
+        invert=False, double_buffer=True, half_duplex=True, display_type=0, asynchronous=False, initialize=True,
+        color_format=lv.COLOR_FORMAT.NATIVE
```

Comment:
`NATIVE` is the default, it's not required to set it explicitly.

Reply:
Replaced with default `None`